zkfacade/pom.xml

```diff
@@ -65,6 +65,19 @@
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.zookeeper</groupId>
+      <artifactId>zookeeper</artifactId>
+      <!-- Need to exclude this artifact, since it exports edu.umd.cs.findbugs.annotations,
```

Comment:
Suggest to emphasize that the exclusion is done on behalf of modules that use zkfacade in provided scope. The exclusion is not really needed here, but then all those modules would have to exclude it.

An alternative could be to re-export the package from zkfacade. I suggest to mention that in the comment as well, for later reference.